node/forcedroid.js

```diff
@@ -106,6 +106,11 @@ function createApp(config) {
         process.exit(8);
     }
 
+    //If no target directory specified, default to current directory
+    if(config.targetdir == null || config.targetdir.length == 0) {
```

Comment:
If `config.targetdir` is not set, let's automatically set it to `appName`. All the other changes in this file can be reverted. I feel like that would be a much cleaner approach.